### PR TITLE
Remove chalk

### PR DESCRIPTION
**Pull request by pluma**
This PR restores chalk v1 and replaces it with `internal.COLORS` in our code where necessary.

It also removes the stray duplicate CHANGELOG entry accidentally added in #14152.

### CODE REVIEW

File: js/common/modules/@arangodb/util.js [hunk 1]
```diff
@@ -111,7 +109,11 @@ exports.codeFrame = function (e, basePath, withColor = internal.COLOR_OUTPUT) {
       const location = `@ ${
         basePath ? ctx.fileName.slice(basePath.length + 1) : ctx.fileName
       }:${ctx.lineNumber}:${ctx.columnNumber}\n`;
-      return (withColor ? chalk.grey(location) : location) + frame;
+      return (
+        withColor
+        ? `${internal.COLORS.COLOR_BRIGHT}${location}${internal.COLORS.COLOR_RESET}`
```
Comment (pluma) [inline js/common/modules/@arangodb/util.js:114]: I'm not sure what our equivalent of bright black (i.e. grey) is but I assume this is it?

**Comment by pluma** [comment]
This PR now simply downgrades chalk to version 1 instead.

**Review by dothebart — APPROVED** [review]

**Review by dothebart — APPROVED** [review]

**Comment by KVS85** [comment]
https://jenkins.arangodb.biz/job/arangodb-ANY-docker-packages/1290/

**Comment by pluma** [comment]
> https://jenkins.arangodb.biz/job/arangodb-ANY-docker-packages/1290/

Green.

PR is red on Linux but that seems an unrelated crash. I'm rebuilding now.

**Comment by KVS85** [comment]
```
Image Digest: sha256:cb4e344378347f77bc21898d918fdb96ae5820424b8c75c07023ec635753bdb3
Full Tag: docker.io/arangodb/arangodb-preview:PR14505
Image ID: 313c206386489064c7151c95c015f0c087d9325428682f00870407de103cb6b3
Status: pass
Last Eval: 2021-07-16T11:00:35Z
Policy ID: 2c53a13c-1765-11e8-82ef-23527761d060
Final Action: warn
Final Action Reason: policy_evaluation

Gate              Trigger            Detail                                                                                     Status
dockerfile        instruction        Dockerfile directive 'HEALTHCHECK' not found, matching condition 'not_exists' check        warn
```
Image Digest: sha256:ea1b2ff12cb9b1c439cc987cbc95932afa86f910f70a156a673747401150262f
Full Tag: docker.io/arangodb/enterprise-preview:PR14505
Image ID: c80da7adc03f6705e5cd3d3847ccbbe2f0c8807f676fdb34fef2f594a0cd2187
Status: pass
Last Eval: 2021-07-16T11:22:59Z
Policy ID: 2c53a13c-1765-11e8-82ef-23527761d060
Final Action: warn
Final Action Reason: policy_evaluation

Gate              Trigger            Detail                                                                                     Status
dockerfile        instruction        Dockerfile directive 'HEALTHCHECK' not found, matching condition 'not_exists' check        warn
```

**Review by KVS85 — APPROVED** [review]
LGTM.

**Pull request merged**